sdk/tables/Azure.Data.Tables/tests/TableServiceClientTests.cs

```diff
@@ -27,6 +28,14 @@ public TableServiceClientTests(bool isAsync) : base(isAsync)
         private readonly Uri _url = new Uri($"https://someaccount.table.core.windows.net");
         private readonly Uri _urlHttp = new Uri($"http://someaccount.table.core.windows.net");
 
+        private TableServiceClient service_Instrumented { get; set; }
+
+        [SetUp]
+        public void TestSetup()
+        {
+            service_Instrumented = InstrumentClient(new TableServiceClient(new Uri("https://example.com")));
```

Comment:
nit: we can use the existing `_url` value for this.

Also, since we are adding this, would you mind refactoring `ServiceMethodsValidateArguments` to use it as well?

Reply:
Not sure what to refactor because the method already uses `_url`.